pex/resolver.py

```diff
@@ -978,13 +978,26 @@ def iter_targets():
                 # Build for the specified local interpreters (on the current platform).
                 yield DistributionTarget.for_interpreter(interpreter)
 
+        def check_platform_against_interpreters(platform):
```

Comment:
Please limit this PR to the description and break this out in a new PR.

Reply:
You got it!